packages/react-storybook-addon/src/decorators/withFluentProvider.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { StoryFn as StoryFunction } from '@storybook/addons';
+
+import { themes, defaultTheme, FluentProvider } from '../theme';
+import { THEME_ID } from '../constants';
+import { FluentGlobals, FluentStoryContext } from '../hooks';
+
+import { makeStyles } from '@fluentui/react-make-styles';
```

Comment:
I'm not sure what 'in general' means here. The addon already uses `FluentProvider` as a dependency.

In any case I've removed the usages of makestyles in this PR

Reply:
> I'm not sure what 'in general' means here.

it means that storybook has its own way of styling (emotion). to follow the API of addons unified solutions should be used instead of custom one especially for public facing addons.

Reply:
> makeStyles is necessary in the FluentProvider since we need a background for stories that can be theme switched. (see #20346 (comment))

If I'm not wrong having this styled via emotion (storybook way) or inline would increase the specificity so there is no need for make-styles being used manually. What the Provider does under the hood is implementation detail and addon should not care about it - thus following generic addons styling solution. Obtaining the color value should be achievable directly from react-theme

Reply:
Ah ok that makes sense, I fetched the direct theme value and removed makestyles in [this commit](https://github.com/microsoft/fluentui/pull/20346/commits/9a54f98eddfbf2cbe7858430bcce491b49837525)